src/Components/PrimitiveValue/PrimitiveValue.js

```diff
@@ -63,10 +63,16 @@ class PrimitiveValue extends React.Component {
   }
 
   render() {
+    if (this.props.value === null) {
```

Comment:
This check is almost the same you're performing few lines below (`isUndefined`). Could you please unify these?

Moreover, a `Primitive` value should always be not null so - these checks should probably happen in the container component, but that's another story